resources/views/checkout/partials/sections/payment.blade.php

```diff
@@ -0,0 +1,55 @@
+<x-rapidez-ct::card.inactive>
+    <form id="payment" class="flex flex-col gap-4" v-on:submit.prevent="save(['payment_method'], 4)">
+        <div v-for="(method, index) in checkout.payment_methods">
+            <x-rapidez-ct::input.radio
+                class="min-h-[40px]"
+                name="payment_method"
+                v-bind:value="method.code"
+                v-bind:dusk="'method-'+index"
+                v-model="checkout.payment_method"
+                required
+            >
+                <div class="text-ct-neutral font-medium max-sm:max-w-[180px]">@{{ method.title }}</div>
```

Comment:
why do you need this max-w?

Reply:
Because the method.title can be long in for example: "iDeal - Online betalen via je Nederlandse bank". So on small screens the payment icon won't fit after the text .
<img width="362" alt="Screenshot 2023-11-30 at 14 59 28" src="https://github.com/rapidez/confira/assets/85165259/a1b207a5-9dfa-4d88-bc01-01a8f4acd107">